core/templates/dev/head/css/oppia.css

```diff
@@ -119,6 +119,56 @@ body {
   margin-top: 15px;
 }
 
+.oppia-profile-tabs md-content {
+  background-color: white;
+}
+
+.oppia-profile-tabs md-content md-tabs {
+  background: #f6f6f6;
+  border: 1px solid #e1e1e1; 
+}
+
+.oppia-profile-tabs md-content md-tabs md-tabs-wrapper {
+  background: white; 
+}
+
+.oppia-profile-tabs md-content h1:first-child {
+  margin-top: 0; 
+}
+
+.oppia-profile-tabs md-tab {
+  background-color: #009688;
+}
+
+md-tabs-ink-bar {
```

Comment:
Can you prefix this with .oppia-profile-tabs, so that it doesn't apply more globally? If we can get rid of the !important, so much the better.